src/main.c

```diff
@@ -4,6 +4,7 @@ vim: noexpandtab:ts=4:sts=4:sw=4
 nudoku
 
 Copyright (C) 2014 - 2020 Michael "jubalh" Vetter - jubalh _a-t_ iodoru.org
+Copyright (C) 2022 Sandroid75 - real-time elapsed playing time
```

Comment:
just because one change?

Reply:
of course you can't delete this comment, it's just a small contribution.